crates/swc/tests/fixture/deno/paths/issue-2724/output/src/index.ts

```diff
@@ -2,8 +2,8 @@
 Object.defineProperty(exports, "__esModule", {
     value: true
 });
-const _interop_require_wildcard = require("@swc/helpers/lib/_interop_require_wildcard.js").default;
+const _interop_require_wildcard = require("@swc/helpers/_/_interop_require_wildcard");
 (async function() {
-    const { displayA  } = await Promise.resolve().then(()=>/*#__PURE__*/ _interop_require_wildcard(require("../packages/a/src/index")));
+    const { displayA  } = await Promise.resolve().then(()=>/*#__PURE__*/ (0, _interop_require_wildcard._interop_require_wildcard)(require("../packages/a/src/index")));
```

Comment:
I think this will regress the bundle size by margin.

Reply:
Do you mean the indirect call?

We known that there is no `this` issue in our helpers.
Is it safe to call function directlly?

Reply:
I meant that `_interop_xxx` cannot be optimized by the minifier.

I think it's better to invoke it directly

Reply:
> I meant that `_interop_xxx` cannot be optimized by the minifier.
> 
> I think it's better to invoke it directly

Done. fixed by keep using the pattern `fn.apply(this, arguments)` .

Reply:
I still need to handle it in CJS/AMD/UMD

Reply:
> I meant that `_interop_xxx` cannot be optimized by the minifier.
> 
> I think it's better to invoke it directly

The first solution is to disable rewriting exported functions in esm so that we can safely use `var foo = require("foo").foo`.
But if other bundler handle our helpers, it does not know that our export is constant and may not apply such optimization.

The second solution is to add another constant named export with shorter name such as `_`, and then use it in transpiled file as follows: 

```JavaScript
var foo = require("foo");
foo._(bar);.
```

Which is better?

Reply:
Wow, the second one sounds cool. What a clever trick!

Reply:
is there a simple way to get helpers SyntaxContext?
I am trying to skip the indrect call `(0, foo)._` for helpers.

Reply:
I got it by using
```
HELPERS.with(|helper| helper.mark());
```